README.md

```diff
@@ -116,34 +116,113 @@ This allow to **configure** the layouts **on the go** while using it.
+... window panes 
+* `Super+d` Focus on the next window pane, left to right.
+* `Super+a` Focus on the previous window pane, right to left.
+
+...workspaces 1 through 10
+* `Super+1`, `Super+2` ... `Super+0` Navigate to specific workspace 1 - 10
+* `Super+s` Focus on the next workspace 1 - 10 (down).
+* `Super+w` Focus on the previous workspace 10 - 1 (up).
+
+...multiple monitors -- Alt+Super+__
+* `Alt+Super+left` Set focus to the left monitor.
+* `Alt+Super+right` Set focus to the right monitor.
+* `Alt+Super+down` Set focus to the downward monitor.
+* `Alt+Super+up` Set focus to the upward monitor.
+
+...to different workspaces and windows via switching applications (gnome key-binding)
+* `Super+tab` Switch between applications
+
+#### Moving a window
+... in the current workspace
+* `Super+left` Move the current window to the left.
+* `Super+right` Move the current window to the right.
+* `Super+up` Move the current window to upper workspace (top).
+* `Super+down` Move the current window to lower workspace (bottom).
+* `Shift+Super+1`, `Shift+Super+2`, ... `Shift+Super+0` Move window to specific workspace  
+
+...to multiple monitors
+* `Shift+Super+a` Move the current window to the monitor on the left.
+* `Shift+Super+d` Move the current window to the monitor on the right.
+* `Shift+Super+w` Move the current window to the upper workspace.
+* `Shift+Super+s` Move the current window to the lower workspace.
+
+... alternate combinations for multiple monitors (gnome keybindings)
+* `Shift+Super+left` Move the current window to the left monitor.
+* `Shift+Super+right` Move the current window to the right monitor.
+* `Shift+Super+down` Move the current window to the downward monitor.
+* `Shift+Super+up` Move the current window to the upward monitor. 
+
+... in combination with the mouse
 * `Super+[MouseDrag]` Move window around.
-* `Super+Shift+A` Move the current window to the left.
-* `Super+Shift+D` Move the current window to the right.
-* `Super+Shift+W` Move the current window to the upper workspace.
-* `Super+Shift+S` Move the current window to the lower workspace.
+
+#### Resize a window
+* `Ctrl+Super+left` Increase width moving border to the left.
+* `Ctrl+Super+up` Increase height moving border upward.
+* `Ctrl+Super+right` Increase width moving border to the right.
+* `Ctrl+Super+down` Increase height moving border downward.
+
 
 #### Extra Hotkeys
-* `Super+Space` Cycle the tiling layout of the current workspace.
+
+* `Shift+Super+Space` Reverse cycle the tiling layout of the current workspace.
 * `Super+Escape` Toggle the UI of Material-shell, like a Zen mode.
 
 # Installation
 
-#### Get it in two clicks
+#### Get it in two clicks (single-user installation, stable release)
 * Navigate to [extensions.gnome.org](https://extensions.gnome.org/extension/3357/material-shell/)
 * Switch the toggle ON
 
-#### Get the most up to date version with Git
+#### Get the most up to date version with Git (multi-user installation, in development)
```

Comment:
Unfortunately it's not true for all the below methods. Some install the stable releases and some the dev branch. We shouldn't make any statement at this time